roofit/roofitcore/inc/RooBinWidthFunction.h

```diff
@@ -25,7 +25,13 @@
 namespace BatchHelpers { struct RunContext; }
 
 class RooBinWidthFunction : public RooAbsReal {
-public:
+  static bool _enabled;
+  
+public:  
+  static void enable();
+  static void disable();
+  static bool enabled();    
```

Comment:
```suggestion
  static void enableClass();
  static void disableClass());
  static bool isClassEnabled();    
```
I would extend the names such that it's clear that this is enabling all instances, i.e. that it's acting on the class level and not the instance level. I suggest some names here, you think they make sense?

Also I would follow the convention that functions that return bool flags are called `is*` here, because `enabled` looks too similar to `enable`.